src/models/Token.ts

```diff
@@ -78,4 +80,11 @@ export class Token {
 
     return asString.slice(0, indexOfSignificantDigit + digits)
   }
+
+  minusEther(amount: string): Token {
```

Comment:
```suggestion
  minusBaseUnits(amount: string): Token {
```
Also does it make sense to have `amount` as `bigint` and `BigNumber` in addition to string.